src/editor.js

```diff
@@ -71,7 +74,7 @@ class MonacoEditor extends React.Component {
     const { language, theme, options } = this.props;
     if (this.containerElement) {
       // Before initializing monaco editor
-      Object.assign(options, this.editorWillMount());
+      Object.assign({}, options, this.editorWillMount());
```

Comment:
Can you explain this change to me?

Reply:
You can mutate `options` object. `options` object is coming from the `this.props` so you could mutate your props.

Reply:
Yeah, I understand that. What I wonder is whether `this.editorWillMount()` should be merged into the options or not. It looks very intentional to me and so I just wondered what was happening here.

Reply:
Let me revert that change since this is not relevant to what I'm trying to fix :)

Reply:
I'd still love to understand it and fix it if necessary. Could you look into it and send a PR if we want to change this?

Reply:
Unfortunately, I didn't write the code and would have to dig into the code myself.

Reply:
Created an issue https://github.com/superRaytin/react-monaco-editor/issues/163